specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/examples/ApiManagementListPolicyFragmentReferences.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2021-04-01-preview",
+    "subscriptionId": "subid",
+    "policyFragmentId": "policyFragment1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.ApiManagement/service/apimService1/policies/policy",
+            "type": "Microsoft.ApiManagement/service/policies",
+            "name": "policy"
+          }
+        ],
+        "count": 1,
```

Comment:
Any special reason to include count? I am not used to seeing a 'count' included in the paged list responses, might be simplest to leave it out.

Reply:
All other APIM APIs return the count, so I'd probably keep it here for consistency.